worldedit-core/src/main/java/com/sk89q/worldedit/LocalConfiguration.java

```diff
@@ -67,6 +67,7 @@
     public int navigationWandMaxDistance = 50;
     public int scriptTimeout = 3000;
     public int calculationTimeout = 100;
+    public int maxCalculationTimeout = 300;
```

Comment:
May want to verify post-config load that this is `>= calculationTimeout`, to prevent weird config setups from existing.